app/views/rubygems/show.html.erb

```diff
@@ -12,7 +12,11 @@
 
 <% if @rubygem.public_versions.count.zero? %>
   <div class="t-body">
-    <p><%= t '.not_hosted_notice' %></p>
+    <% if @rubygem.pushable? %>
+      <p><%= t '.not_hosted_notice' %></p>
+    <% else %>
+      <p><%= (t '.reserved_namespace', contact: link_to('Contact', 'http://help.rubygems.org/')).html_safe %></p>
```

Comment:
> just nitpick

Appreciate it.
![tumblr_inline_mus80klotk1r4zbjb](https://cloud.githubusercontent.com/assets/7680662/17038434/52f695fe-4fb3-11e6-9145-9d7d77982099.gif)